app/config/collections.php

```diff
@@ -4471,6 +4471,39 @@
                 'array' => false,
                 'filters' => [],
             ],
+            [
+                '$id' => ID::custom('status'),
```

Comment:
This is what we use in other collections instead:

```
            [
                '$id' => ID::custom('enabled'),
                'type' => Database::VAR_BOOLEAN,
                'signed' => true,
                'size' => 0,
                'format' => '',
                'filters' => [],
                'required' => true,
                'array' => false,
            ],
```

I would rename to `enabled`